client/components/PrivateOrPublic.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)